sql/core/src/test/resources/sql-tests/inputs/udf/udf-group-analytics.sql

```diff
@@ -0,0 +1,63 @@
+-- This test file was converted from group-analytics.sql.
+CREATE OR REPLACE TEMPORARY VIEW testData AS SELECT * FROM VALUES
```

Comment:
@skonto, sorry last nit. Can you add a comment, with the JIRA you're going to file at https://github.com/apache/spark/pull/25098#discussion_r305313127 ?

e.g.

```
-- TODO: UDF should be inserted and tested at GROUP BY clause after SPARK-XXXXX
```

It's kind of sad that we can't test `GROUP BY` with UDF at grouping specific test cases ...

Reply:
Or you can make one test failed, skip, and similarly comment on it like #25098 for now

Reply:
sure im in the process of doing it let me check, have some connection issues here so need some time

Reply:
@HyukjinKwon I added a top level comment as you mentioned.